django_slack/logs.py

```diff
@@ -0,0 +1,84 @@
+import copy
+import logging
+from . import slack_message
+from django.conf import settings
+from django.views.debug import ExceptionReporter
+
+
+class SlackExceptionHandler(logging.Handler):
+    """
+    An exception log handler that sends log entries to a Slack channel.
+    """
+
+    def __init__(self, **kwargs):
+        # pop any kwargs that shouldn't be passed into the slack message attachment here.
```

Comment:
Still don't really understand this. I mean, I understand the *words* but in about a week I will have no idea why it (what?) is being passed in. And why it shouldn't be passed to the attachment.

Also, slack → Slack :)